app/app.go

```diff
@@ -667,6 +669,7 @@ func NewApp(
 		&app.mintKeeper,
 		&app.kavadistKeeper,
 		app.stakingKeeper,
+		authtypes.NewModuleAddress(govtypes.ModuleName), // Authority
```

Comment:
This is used in quite a few places now -- could we move to a named parameter? `govAuthorityAddr` or similar and reuse?